src/common/items/VkItem.ts

```diff
@@ -19,4 +19,5 @@ export const VkItem = {
   UNSTACKABLE:  Material.GOLDEN_HORSE_ARMOR,
   COLORABLE:    Material.LEATHER_HORSE_ARMOR,
   SEED:         Material.PUMPKIN_SEEDS,
+  PAPER:        Material.PAPER,
```

Comment:
Maybe just don't use VkItem for this one, all paper items are defined in single file.

Reply:
Laetta said to do so.

Reply:
Using VkItem makes /customitem command easier, because it autocompletes, but this is optional